samples/nextjs/src/lib/sitemap-fetcher/plugins/disconnectedSitemapService.ts

```diff
@@ -0,0 +1,52 @@
+﻿import { DisconnectedSitemapService } from '@sitecore-jss/sitecore-jss-nextjs';
+// #START_EMPTY
```

Comment:
We can forget about the `--empty` support at this point since that is going away once we absorb these to the new initializers work (i.e. the disconnected plugin will simply be added to our new post-initializer for styleguide/disconnected).

Let's remove the `#START/END_EMPTY` comments.